pints/toy/_stochastic_degradation_model.py

```diff
@@ -0,0 +1,90 @@
+#
+# Logistic toy model.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+import scipy
+from scipy.interpolate import interp1d
+import pints
+
+from . import ToyModel
+
+
+class StochasticDegradationModel(pints.ForwardModel, ToyModel):
+
+    """
+    Stochastic decay model of a single chemical reaction [1].
```

Comment:
Ok, can we have a bit more information here. For example, reproducing the A -> phi reaction equation from the paper.